cmd/jet/main.go

```diff
@@ -243,10 +252,16 @@ func genTemplate(dialect jet.Dialect, ignoreTables []string, ignoreViews []strin
 		return strslice.Contains(ignoreEnums, strings.ToLower(enum.Name))
 	}
 
+	model := template.DefaultModel()
+
+	if modelPkg != "" {
+		model = model.UsePath("/" + modelPkg)
+	}
```

Comment:
If you add default flag values above, this can be simplified:
```golang
template.DefaultModel().UsePath(modelPkg) // I believe there is no need for "/"
```

Reply:
Noted.

Reply:
There is indeed no need for the `/` as you noted as all `Path` are being used with `path.Join()`